ui/src/styles.css

```diff
@@ -16,30 +16,42 @@
     src: url('/assets/fonts/Gotham Medium.otf');
 }
 
+body {
+  background-color: #f1f2f2;
```

Comment:
As a heads up, the background color is actually changing in my sign-up revamp PR, from Lou's feedback. (It is going to be a darker grey than it currently is)